selenium/android/Dockerfile

```diff
@@ -64,26 +65,27 @@ ARG SDCARD_SIZE="500"
 ARG USERDATA_SIZE="500"
 
 RUN \
-	curl -o sdk-tools.zip https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip && \
-	mkdir -p /opt/android-sdk-linux /root/.config/Android\ Open\ Source\ Project/ && \
-	unzip -q sdk-tools.zip -d /opt/android-sdk-linux && \
+	curl -o sdk-tools.zip https://dl.google.com/android/repository/commandlinetools-linux-7583922_latest.zip && \
+	mkdir -p /opt/cmdline-tools /root/.config/Android\ Open\ Source\ Project/ && \
+	unzip -q sdk-tools.zip -d /opt/cmdline-tools && \
+    mv /opt/cmdline-tools/cmdline-tools /opt/cmdline-tools/tools &&\
 	rm sdk-tools.zip && \
 	yes | sdkmanager --licenses
 
 RUN \
-	sdkmanager "emulator" "tools" "platform-tools" "$BUILD_TOOLS" "platforms;$PLATFORM" "$EMULATOR_IMAGE" && \
+	sdkmanager "tools" "platform-tools" "$BUILD_TOOLS" "platforms;$PLATFORM" "$EMULATOR_IMAGE" && \
 	mksdcard "$SDCARD_SIZE"M sdcard.img && \
 	echo "no" | ( \
-	    ([ -n "$ANDROID_DEVICE" ] && avdmanager create avd -n "$AVD_NAME" -k "$EMULATOR_IMAGE" --abi "$ANDROID_ABI" --device "$ANDROID_DEVICE" --sdcard /sdcard.img ) || \
-	    avdmanager create avd -n "$AVD_NAME" -k "$EMULATOR_IMAGE" --abi "$ANDROID_ABI" --sdcard /sdcard.img \
+	    ([ -n "$ANDROID_DEVICE" ] && avdmanager create avd -n "$AVD_NAME" -k "$EMULATOR_IMAGE" --abi "$ANDROID_ABI" --device "$ANDROID_DEVICE" --sdcard sdcard.img ) || \
+	    avdmanager create avd -n "$AVD_NAME" -k "$EMULATOR_IMAGE" --abi "$ANDROID_ABI" --sdcard sdcard.img \
     ) && \
 	ldconfig && \
 	( \
 	    resize2fs /root/.android/avd/$AVD_NAME.avd/userdata.img "$USERDATA_SIZE"M || \
-	    /opt/android-sdk-linux/emulator/qemu-img resize -f raw /root/.android/avd/$AVD_NAME.avd/userdata.img "$USERDATA_SIZE"M \
+	    /opt/emulator/qemu-img resize -f raw /root/.android/avd/$AVD_NAME.avd/userdata.img "$USERDATA_SIZE"M \
```

Comment:
Don't get where `emulator` directory comes from.

Reply:
> Don't get where `emulator` directory comes from.

During building process i saw interesting logs:
=> => # Warning: Package "com.android.repository.impl.generated.v2.RemotePackage@863816ad" (emulator) should be installed in
=> => # "/opt/emulator" but
=> => # it already exists.
=> => # Installing in "/opt/emulator-2" instead.

And turned out that when you fetch system image e.g. sdkmanager "system-images;android-31;default;x86_64"

it automatically fetch emulator directory too.
So i decided remove fetch emulator directly

Reply:
> Don't get where `emulator` directory comes from.

During building process i saw interesting logs:
`=> => # Warning: Package "com.android.repository.impl.generated.v2.RemotePackage@863816ad" (emulator) should be installed in`
`=> => # "/opt/emulator" but`
`=> => # it already exists.`
`=> => # Installing in "/opt/emulator-2" instead.`

And turned out that when you fetch system image e.g. `sdkmanager "system-images;android-31;default;x86_64"`

it automatically fetch emulator directory too.
So i decided remove fetch emulator directly

Reply:
Will these changes work for older Android versions?

Reply:
Interesting question)
What boundaries of supported versions need to test?
Because i'm not sure about backward compatibility of new SDK)

Reply:
@dbudim at least Android 6.0+.

Reply:
ok, i'll check 6th and back with result)

Reply:
i've checked 6,7 versions - doesn't work, seems problem in vulkan drivers. 
Need some free time for investigate this. Will do a bit later)

Reply:
@dbudim sure